bootstrap.cmd

```diff
@@ -145,15 +145,39 @@ echo Tools can be found at http://llvm.org/releases/download.html#3.8.0
 :: Download clang-format and clang-tidy
 echo Downloading formatting tools
 
-call powershell Invoke-WebRequest -Uri "https://clrjit.blob.core.windows.net/clang-tools/windows/clang-format.exe" -OutFile bin\clang-format.exe
-if errorlevel 1 echo ERROR: failed to download clang-format.&set __ExitCode=1&goto :eof
+call :download_url clang-format "https://clrjit.blob.core.windows.net/clang-tools/windows/clang-format.exe" bin\clang-format.exe
+if %__ExitCode% NEQ 0 goto :eof
 
-call powershell Invoke-WebRequest -Uri "https://clrjit.blob.core.windows.net/clang-tools/windows/clang-tidy.exe" -OutFile bin\clang-tidy.exe
-if errorlevel 1 echo ERROR: failed to download clang-tidy.&set __ExitCode=1&goto :eof
+call :download_url clang-tidy "https://clrjit.blob.core.windows.net/clang-tools/windows/clang-tidy.exe" bin\clang-tidy.exe
+if %__ExitCode% NEQ 0 goto :eof
```

Comment:
This doesn't change previous behavior, so I'm not saying this has to fix it. But it's not clear to me who uses `__ExitCode` for this subprocess and how that will bubble up to the caller.

Reply:
This is the `:build_jitutils` path, and the caller uses `__ExitCode` to set the exit code by calling `:script_exit`